client/packages/requisitions/src/ResponseRequisition/api/hooks/utils/useIsRemoteAuthorisation.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  RequisitionNodeApprovalStatus,
+  useAuthContext,
+} from '@openmsupply-client/common';
+import { useResponse } from '..';
+
+export const useIsRemoteAuthorisation = () => {
+  const { store } = useAuthContext();
+
+  const { approvalStatus } = useResponse.document.fields(['approvalStatus']);
+  const authoriseResponseRequisitions =
```

Comment:
This is slightly different from `useIsRemoteAuthorisation` for request requisition, which doesn't check the preference (I think, technically we just need to check approvalStatus (to figure out if extra columns or approvalStatus needs to be displayed in detail view)